omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/OrderGroupResource1_12.java

```diff
@@ -0,0 +1,214 @@
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addRequiredProperty("patient");
+		description.addRequiredProperty("encounter");
+		description.addProperty("orders");
+		description.addProperty("orderSet");
+		return description;
+	}
+	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		}
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		} else if (rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("auditInfo", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderGetRef")))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+		}
+		else if (rep instanceof RefRepresentation) {
+			modelImpl
+			        .property("display", new StringProperty())
+			        .property("uuid", new StringProperty());
+			
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation representation) {
+		ModelImpl model = new ModelImpl()
+		        .property("patient", new StringProperty().example("uuid"))
+		        .property("encounter", new StringProperty().example("uuid"))
+		        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")))
+		        .property("orderSet", new StringProperty().example("uuid"));
+		return model;
+	}
+	
+	@PropertySetter("orders")
+	public static void setGroupMembers(OrderGroup orderGroup, List<Order> members) {
+		for (Order member : members) {
```

Comment:
I think a null check on 'members' would be nice.

Reply:
ya , thats true @HelioStrike

Reply:
actaully that propert setter wasnt even necesary. it was already implemented by the default setter and getters in the OrderGroup class